### DISCUSSION TITLE
How to use a custom slide

### DISCUSSION THREAD

**Discussion by Maldino** [AppIntro/AppIntro#1018] [Q&A] [answered]
Hello,

I am new and want to add a custom slide .. I tried both of that

`addSlide(AppIntroCustomLayoutFragment.newInstance(R.layout.activity_one));`

that causes crash

`AppIntroCustomLayoutFragment.newInstance(R.layout.activity_one);`

that code show no thing

**Comment by cortinico** [comment, marked as answer]
Please find an example here:
https://github.com/AppIntro/AppIntro/blob/0b782dbcfe0c4f0fa43b0b87bc736852c5e1ab40/example/src/main/java/com/github/appintro/example/ui/custom/CustomLayoutIntro.kt#L9-L22

> addSlide(AppIntroCustomLayoutFragment.newInstance(R.layout.activity_one));
> that causes crash

Which crash?

**Reply by Maldino** [reply]
> > addSlide(AppIntroCustomLayoutFragment.newInstance(R.layout.activity_one));
> > that causes crash
> 
> Which crash?

Thanks for reply ...
I found the reason of the crash .  That was because i used color transitions between slides - which not working with custom slides - and i solved it by removing this line  `setColorTransitionsEnabled(true);`
